packages/office-ui-fabric-react/src/components/Toggle/Toggle.styles.ts

```diff
@@ -79,28 +79,28 @@ export const getStyles = memoizeFunction((
     },
 
     thumb: {
-      width: '.5em',
```

Comment:
Even after adding "box-sizing:border-box", the thumb was not getting filled completely:
![image](https://user-images.githubusercontent.com/6390909/27119885-3a34017a-50ff-11e7-9351-24fc99797631.png)

So, anyway we will be needing the border width